desktop-environment.el

```diff
@@ -176,6 +193,17 @@ portion of the screen."
   "Shell command toggling bluetooth."
   :type 'string)
 
+
+;;; Helper variables - keyboard layout
+
+(defvar desktop-environment-keyboard-layouts-ring []
+  "Varible for storing whole commands for setxkbmap.
```

Comment:
what do you mean by "whole commands"? According to the code, only the arguments are stored.

What about

> Store a ring view of `desktop-environment-keyboard-layouts` elements.

Reply:
It is residual code from before when I was concating "setxkbmap -layout" on the whole list. I am going to remove it.